.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Pipedrive Wrapper
+
+on:
+  pull_request:
+    branches:
+      - "*"
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: Ruby ${{ matrix.ruby_version }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        ruby_version: ["2.7.8", "3.0.6"]
```

Comment:
Can you make the versions like `2.7` and `3.0` please?

Reply:
sure, I made this because we use those version of ruby, feel free to suggest me some changes (having more versions to test in some folks use 3.1 for example)

Reply:
Is it ok for you like this @simonoff ?